<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="digitBotton7">
       <property name="text">
        <string>7</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="divisionButton">
       <property name="text">
        <string>%</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">multOrDivGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="equalButton">
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="digitBotton2">
       <property name="text">
        <string>2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="digitBotton3">
       <property name="text">
        <string>3</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="digitBotton6">
       <property name="text">
        <string>6</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="subtractionButton">
       <property name="text">
        <string>-</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">addOrSubGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="multipleButton">
       <property name="text">
        <string>x</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">multOrDivGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="digitBotton0">
       <property name="text">
        <string>0</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="additionButton">
       <property name="text">
        <string>+</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">addOrSubGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="digitBotton1">
       <property name="text">
        <string>1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="digitBotton9">
       <property name="text">
        <string>9</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="digitBotton4">
       <property name="text">
        <string>4</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="digitBotton5">
       <property name="text">
        <string>5</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="digitBotton8">
       <property name="text">
        <string>8</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">digitButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="digitButtonGroup"/>
  <buttongroup name="addOrSubGroup"/>
  <buttongroup name="multOrDivGroup"/>
 </buttongroups>
</ui>
